File: project/src/com/lasthopesoftware/bluewater/servers/library/items/media/files/BaseMenuViewHolder.java
package com.lasthopesoftware.bluewater.servers.library.items.media.files;

import android.widget.ImageButton;

public class BaseMenuViewHolder {
	public BaseMenuViewHolder(final ImageButton viewFileDetailsButton, final ImageButton playButton) {
		this.viewFileDetailsButton = viewFileDetailsButton;
		this.playButton = playButton;
	}
	
	public final ImageButton viewFileDetailsButton;
	public final ImageButton playButton;
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/browsing/items/media/files/BaseMenuViewHolder.kt
package com.lasthopesoftware.bluewater.client.browsing.items.media.files

import android.widget.ImageButton
import com.lasthopesoftware.bluewater.shared.android.view.LazyViewFinder

open class
BaseMenuViewHolder(
	private val viewFileDetailsButtonFinder: LazyViewFinder<ImageButton>,
	private val playButtonFinder: LazyViewFinder<ImageButton>)
{
	val viewFileDetailsButton: ImageButton
		get() = viewFileDetailsButtonFinder.findView()

	val playButton: ImageButton
		get() = playButtonFinder.findView()
}
